Alonso/add lx examples

elv1s opened the pull request:
adding examples

iapplx examples ("hello world", "basic pool")
icontrollx examples ("parrot worker")

elv1s committed:
adding icontrollx and iapplx examples

elv1s committed:
update readme

manjubhat commented on iapplx/f5-iAppsLX-basic-pool/package.json:
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "f5-iappslx-basic-pool",
+  "version": "1.0.0",
```
I think we should use the same version as in RPM spec file

manjubhat commented on iapplx/f5-iAppsLX-basic-pool/README.md:
```diff
@@ -0,0 +1,100 @@
+
+## Build (requires rpmbuild)
+
+    $ ./scripts/build
```
Update RPM build instructions

manjubhat commented on iapplx/f5-iAppsLX-basic-pool/Dockerfile:
```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
```
I think this dockerfile is not used. Could you remove this

elv1s committed:
address pull request comments
- update basic-pool readme build instructions
- update basic-pool package version
